### PR TITLE
 Platform feature matrix 

### PR DESCRIPTION
**Pull request by enkore**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -117,6 +117,70 @@ the old version using the same steps as shown above.
...
++-------------------+----------+----------+-----------+------------+
+| Windows (cygwin)  | Manually | No [4]_  | No        | No         |
++-------------------+----------+----------+-----------+------------+
+
+Some Distributions (e.g. Debian) run additional tests after each release, these
+are not reflected here.
+
+Other Unix-like operating systems may work as well, but have not been tested at all.
+
+Note that most of the platfrom-dependent features also depend on the file system.
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: typo "platform"

File: docs/installation.rst [hunk 2]
```diff
@@ -117,6 +117,70 @@ the old version using the same steps as shown above.
...
+| NetBSD            | Manually | No [5]_  | No [7]_   |            |
++-------------------+----------+----------+           +------------+
+| Solaris 11        | Yes [2]_ | No       |           | n/a        |
++-------------------+----------+----------+           +------------+
+| OpenIndiana       | No       | No       |           | n/a        |
++-------------------+----------+----------+-----------+------------+
+| Windows (native)  | Yes [3]_ | Yes      | ?         | ?          |
++-------------------+----------+----------+-----------+------------+
+| Windows (cygwin)  | Manually | No [4]_  | No        | No         |
++-------------------+----------+----------+-----------+------------+
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: some of these don't really fit under headline "supported platforms", rather "other platforms that might work".

File: docs/installation.rst [hunk 3]
```diff
@@ -117,6 +117,70 @@ the old version using the same steps as shown above.
...
+.. [#tst]
+    *Manually* means that the test suite is run manually e.g. before a release -
+    but may be run even less frequently than that.
+.. [#acls] The native access control list mechanism of the OS. This normally limits access to
+    non-native ACLs. For example, NTFS ACLs aren't completely accessible on Linux with ntfs-3g.
+.. [#xattr] extended attributes; key-value pairs attached to a file, mainly used by the OS.
+    This includes resource forks on Mac OS X.
+.. [#flags] aka *BSD flags*. The Linux set of flags [6]_ is portable across platforms.
+    The BSDs define additional flags.
+
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: As this is not just about platforms, but also feature support, this maybe should be linked together or merged with the other paragraph that talks about which fs features we support.

Comment (enkore) [inline docs/installation.rst]: good idea, that one is a bit lost in the FAQ anyway

**Comment by enkore** [comment]
WIP due to ongoing changes in CI setup.

File: docs/installation.rst [hunk 4]
```diff
@@ -117,6 +117,91 @@ the old version using the same steps as shown above.
...
+Besides regular file and directory structures, |project_name| can preserve
+
+    * Hardlinks (considering all files in the same archive)
+    * Symlinks (stored as symlink, the symlink is not followed)
+    * Special files:
+
+        * Character and block device files
+        * FIFOs ("named pipes")
+        * Special file (e.g. block devices, mappers) *contents* can be
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: mappers? see is_special().

**Comment by ThomasWaldmann** [comment]
looks good. if you do not collapse it away anyway, maybe replace the (not yet) commit comment by something more descriptive.

**Comment by enkore** [comment]
Ok, I'll merge it when I'm done with the CI stuff :)

File: docs/installation.rst [hunk 5]
```diff
@@ -117,6 +117,84 @@ the old version using the same steps as shown above.
...
+| NetBSD           |            | n/a      | No [2]_   |            |
++------------------+            +----------+-----------+------------+
+| Solaris 11       |            | No [3]_              | n/a        |
++------------------+            |                      |            |
+| OpenIndiana      |            |                      |            |
++------------------+------------+----------+-----------+------------+
+| Windows (native) | AppVeyor_  | Yes      | ?         | ?          |
++------------------+------------+----------+-----------+------------+
+| Windows (cygwin) | Manually   | No [4]_  | No        | No         |
++------------------+------------+----------+-----------+------------+
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: Hmm, maybe the test sites could go into a separate section and just keep the supported features here?

**Comment by enkore** [comment]
merge?

**Comment by ThomasWaldmann** [comment]
a lot of this applies to 1.0-maint also. if we apply just to master, docs diverge more.

so, apply against 1.0-maint (slightly modified for correctness), merge into master, update it with master differences?

**Pull request merged**